src/components/action-addtion.vue

```diff
@@ -0,0 +1,41 @@
+<template>
```

Comment:
Rename the file into `add-action-control.vue`, as the file is actually handle the html element for adding a new action to the selected operation.